freenet-ext/build.xml

```diff
@@ -251,22 +256,27 @@
 
 	<macrodef name="lib-report">
 		<attribute name="lib" />
-		<sequential>
-			<if>
-				<istrue value="${extlib.@{lib}.suppress}"/>
-				<then><echo message="suppress @{lib} is ON"/></then>
-			</if>
+		<sequential if:true="${extlib.@{lib}.suppress}">
+			<echo message="suppress @{lib} is ON"/>
+		</sequential>
+		<sequential unless:true="${extlib.@{lib}.suppress}">
+			<!-- To preserve the old behaviour of this macro, I've commented this out.
+			<echo message="suppress @{lib} is NOT ON"/>
```

Comment:
Under freenet-ext/build.properties are the extlib.@{lib}.suppress variables. These are set to true or false. A message to a developer saying "suppress lzma is OFF" is outright confusing. Not only is it a double negative, but it does not describe the behavior of the suppress variable.
It seems more useful to say NOT ON simply because there is less confusion about what that means. With ant conditionals we don't check to see if extlib.${lib}.suppress is OFF or FALSE, we only check to see if it is true or if it is not true. Sending a message saying it is OFF suggests to the developer that there is a difference between extlib.${lib}.suppress being set but not true, and extlib.${lib}.suppress not being set at all. As the current design of the build script is right now, there is no such distinction.
To make my point, imagine you modify your build.properties as follows:
![extlib wrapper suppress eq banana](https://cloud.githubusercontent.com/assets/10847317/6140552/ecbfbc3c-b15e-11e4-891e-af78a90117e1.PNG)
If we use your convention of saying unless:true -> suppress ${lib} is OFF then we see the following:
![extlib wrapper suppress eq banana off](https://cloud.githubusercontent.com/assets/10847317/6140564/1f99dffc-b15f-11e4-9f0c-da3cef718501.PNG)
This just isn't right. extlib.wrapper.suppress does not equal OFF or FALSE or anything that is a synonym with FALSE. Calling it OFF does nothing except for mislead the developer who is to see the message.
If the wording is NOT ON then the output reflects that extlib.wrapper.suppress does not equal TRUE. We don't want to make a statement to the person compiling this application about whether or not suppression is OFF because we only check to see whether or not it is ON.